sockets/overlay.js

```diff
@@ -39,6 +39,18 @@ module.exports = function (io) {
       overlay.emit('play intro', msg);
       console.log('play intro: ' + JSON.stringify(msg));
     });
+
+    socket.on('change layout', function(msg) {
+      client.set(redisKey, JSON.stringify(msg));
+      overlay.emit('change layout', msg);
+      console.log('change layout: ' + JSON.stringify(msg));
+    });
+
+    socket.on('obs connect', function(msg) {
+      client.set(redisKey, JSON.stringify(msg));
```

Comment:
This will override the other overlay data in redis, so you'd probably want a different key here.